sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -2215,51 +2215,152 @@ public void testZetaSQLNestedQueryFive() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  /////////////////////////////////////////////////////////////////////////////
+  // TIMESTAMP type tests
+  /////////////////////////////////////////////////////////////////////////////
+
   @Test
-  @Ignore("Does not support DATE_ADD and TIME_ADD.")
-  public void testDateAndTimeAddSub() {
```

Comment:
This ignored test got removed in this cleanup PR. But I will add it back and make it working soon once we support DATE and TIME types.